tests/integration/expected_test_results/web_html/user-register.html

```diff
@@ -1639,33 +1620,12 @@ <h3 class="title-5 text-medium">Discover the project</h3>
 <script src="//static.openfoodfacts.localhost/js/dist/display.js"></script>
 <script src="//static.openfoodfacts.localhost/js/dist/stikelem.js"></script>
 <script src="//static.openfoodfacts.localhost/js/dist/scrollNav.js"></script>
+<script src="//static.openfoodfacts.localhost/js/dist/togglepassword.js"></script>
+<script src="//static.openfoodfacts.localhost/js/dist/user_form.js"></script>
 
 <script>
 $(function() {
 
-$(document).ready( function () {
```

Comment:
I think this should not have been removed: when I check the "this is a pro account" checkbox, I'm not shown the warning anymore:

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/fce759f7-a7df-498f-9699-a53c40f52d8c)

This is what we have in production:

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/40be2901-ad0a-4848-b17b-6c5c5cdd7942)